src/doc/trpl/compound-data-types.md

```diff
@@ -196,8 +196,8 @@ Now, we have actual names, rather than positions. Good names are important,
 and with a struct, we have actual names.
 
 There _is_ one case when a tuple struct is very useful, though, and that's a
-tuple struct with only one element. We call this a *newtype*, because it lets
-you create a new type that's similar to another one:
+tuple struct with only one element. We call this the *newtype* pattern, because
+it lets you create a new type that adds semantic meaning to another one:
```

Comment:
Yeah, I was trying to find a concise change to explicitly suggest *why* this could be valuable, in light of the given `Inches(i32)` example ("lets you create a new type that's similar to another one" is kind of just empty, dead verbiage...), but not sure that my edit has quite succeeded. Emphasis was meant to be on "semantic meaning" without any suggestion of behavioral significance. I'll try to come up with a better way to get this across. Maybe the example makes it clear enough without any need to say it, but "similar to another one" still needs to find the chopping block somehow IMO.